snakes2.py

```diff
@@ -0,0 +1,150 @@
+# coding=utf-8
```

Comment:
Why is there another snakes solution file here? I'm not going to review both - pick one and work on it together